CONTRIBUTING.md

```diff
@@ -36,7 +33,7 @@ Now you are ready to contribute!
 First, [set up your development environment](#setting-up-a-development-environment).
 
 Our tests are managed by `tox`, which runs both `pytest` tests and our system of makefiles.
-The regression does not end on the first failure, but continues until all tests in the `test` and `example` directories have been run.
+The regression does not end on the first failure, but continues until all tests in the `/tests` and `/examples` directories have been run.
```

Comment:
If tests fail the examples are not run (https://github.com/cocotb/cocotb/runs/1301183158?check_suite_focus=true)

Reply:
Yeah that's a bug that's been around a while. I'd rather not report that as a fact.

Reply:
We also don't (or at least used to) run all examples in regression, but I don't think that level of details needs to be mentioned here.